sabnzbd/cfg.py

```diff
@@ -278,6 +278,7 @@ def validate_server(value):
 x_frame_options = OptionBool("misc", "x_frame_options", True)
 require_modern_tls = OptionBool("misc", "require_modern_tls", False)
 num_decoders = OptionNumber("misc", "num_decoders", 3)
+deobfuscate_final_filenames = OptionBool("misc", "deobfuscate_final_filenames", False)
```

Comment:
We shouldn't forget to add it to the Config 😄

Reply:
done